playbooks/tasks/iworx-php-scl.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Enable Interworx SCL PHP Support
+  command: >
+    nodeworx
+    -un
+    -c Http
+    -a enableMultiplePhp
+
+- name: Find Installed PHP Versions
+  find:
+    paths: "{{ php_remi_root }}"
+    patterns: "php*"
+    file_type: "directory"
+  register: php_dir_names
+
+- name: Enable PHP Versions + Set Default Version
+  command: >
+    nodeworx
+    -un
+    -c Http
+    -a multiplePhpOptions
+    --enabled_php_versions "{{ item.path }}"
+    --default_php_version "{{ item.path }}"
+  with_items: "{{ php_dir_names.files }}"
```

Comment:
Just making sure you saw my prev comment on not relying on the random sort order of the 'find' command.  If you can't coax find to actually sort then use command with 'ls' to ensure idempotency and consistency.